serial/port_linux.go

```diff
@@ -47,6 +47,35 @@ func (p *Port) SetDeadline(t time.Time) error {
 	return nil
 }
 
+// Get the port's DTR pin state
+func (p *Port) DTR() (bool, error) {
+	var status int
+	err := ioctl(unix.TIOCMGET, p.f.Fd(), uintptr(unsafe.Pointer(&status)))
+	if err != nil {
+		return false, err
+	}
+	if status&unix.TIOCM_DTR > 0 {
+		return true, nil
+	}
+	return false, nil
+}
+
+// Set the port's DTR pin state
+func (p *Port) SetDTR(state bool) error {
+	// todo(ahollist): Implement
```

Comment:
what does this comment mean? this looks implemented

Reply:
Oops, yep that's mean to be removed

Reply:
removed in 4d64207